apisix/schema_def.lua

```diff
@@ -437,6 +437,7 @@ local upstream_schema = {
         type = {
             description = "algorithms of load balancing",
             type = "string",
+            default = "roundrobin",
```

Comment:
Do we need to update the doc too?

Reply:
+1. cc @Hazel6869

Reply:
I have done. pls see description. @spacewander @tokers

Reply:
Could you please put the doc in the same PR next time?
So people don't need to check another one.

Reply:
> 

ok